### PR TITLE
Ignore new TS 2.0 unsupported options

### PR DESCRIPTION
**Pull request by emilianosantucci**
With this patch, `typedoc` will build docs also in TS 2.x projects without errors.

**Warning**: actually this is only a workaround to allow `typedoc` to get out docs but it need to improve in internal logic to handle new TypeScript features.

### CODE REVIEW

**Comment by christopherthielen** [comment]
Note that your PR automatically updates as you push commits to the branch.  You'll probably have to redo the original  PR

**Comment by emilianosantucci** [comment]
@christopherthielen, thanks for suggestion. Looking into linked typedoc plugins, I've also notice the usage of typedoc source code but it was not exported on NPM package so I add it.
If the last commit can be confusing about PR target, I can redo and post a new PR after that.

**Comment by blakeembrey** [comment]
Please don't check in built files, they are ignored for a reason. I'll review once you remove them.

**Comment by emilianosantucci** [comment]
@blakeembrey I have fix it.

File: package.json [hunk 1]
```diff
@@ -56,6 +56,7 @@
     "bin",
     "lib",
     "tasks",
+    "src",
```
Comment (blakeembrey) [inline package.json]: Please remove this.

Comment (emilianosantucci) [inline package.json]: Done

**Comment by unsafecode** [comment]
Do we have an estimation for the release of this fix? Thanks

**Comment by jonaskello** [comment]
+1 Could we please have a release with this merged so we can use Typedoc in TS 2.0?

**Comment by christopherthielen** [comment]
https://github.com/TypeStrong/typedoc/releases/tag/v0.5.1

This was released last week.

**Comment by jonaskello** [comment]
@christopherthielen 👍  Thanks, I didn't notice that :-)

**Pull request merged**